### ISSUE TITLE
Cannot autostart module on Linux

### ISSUE THREAD

**Issue by gladykov** [RundownRhino/aw-watcher-mpv-sender#3]
Hey, have some problem, seems module does not wait for connection properly:

I installed it in 
`/opt/activitywatch/aw-watcher-mpv/aw-watcher-mpv`

and created symlink from `/usr/binaw-watcher-mpv`

when I start ActivityWatcher from command line:
`aw-qt` (`aw-watcher-mpv` is added to to autostart_modules in `aw-qt.toml`) I get error:

```
2022-12-18 20:26:45 [ERROR]: Unhandled exception  (root:51)
Traceback (most recent call last):
  File "urllib3/connection.py", line 174, in _new_conn
  File "urllib3/util/connection.py", line 95, in create_connection
  File "urllib3/util/connection.py", line 85, in create_connection
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "urllib3/connectionpool.py", line 703, in urlopen
  File "urllib3/connectionpool.py", line 398, in _make_request
  File "urllib3/connection.py", line 239, in request
  File "http/client.py", line 1282, in request
  File "http/client.py", line 1328, in _send_request
  File "http/client.py", line 1277, in endheaders
  File "http/client.py", line 1037, in _send_output
  File "http/client.py", line 975, in send
  File "urllib3/connection.py", line 205, in connect
  File "urllib3/connection.py", line 186, in _new_conn
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7f763d3d2d70>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "requests/adapters.py", line 489, in send
  File "urllib3/connectionpool.py", line 787, in urlopen
  File "urllib3/util/retry.py", line 592, in increment
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=5600): Max retries exceeded with url: /api/0/buckets/aw-watcher-mpv-curplaying_jakub-aspirevx5591g (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f763d3d2d70>: Failed to establish a new connection: [Errno 111] Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "aw_watcher_mpv_sender/__main__.py", line 3, in <module>
  File "aw_watcher_mpv_sender/main.py", line 30, in main
  File "aw_watcher_mpv_sender/main.py", line 34, in run_with
  File "<string>", line 10, in __init__
  File "aw_watcher_mpv_sender/sender.py", line 36, in __post_init__
  File "aw_watcher_mpv_sender/sender.py", line 125, in create_bucket_once
  File "aw_client/client.py", line 279, in create_bucket
  File "aw_client/client.py", line 43, in g
  File "aw_client/client.py", line 117, in _post
  File "requests/api.py", line 115, in post
  File "requests/api.py", line 59, in request
  File "requests/sessions.py", line 587, in request
  File "requests/sessions.py", line 701, in send
  File "requests/adapters.py", line 565, in send
requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=5600): Max retries exceeded with url: /api/0/buckets/aw-watcher-mpv-curplaying_jakub-aspirevx5591g (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f763d3d2d70>: Failed to establish a new connection: [Errno 111] Connection refused'))
```

This will not hapen when I first start ActivityWatcher separately , and then will run `aw-watcher-mpv`.
Same error will happen when I run `aw-watcher-mpv` without starting the server.

So this is why I think watcher should first wait for server

**Comment by gladykov** [comment]
So seems official watcher is doing crude sleep:
https://github.com/ActivityWatch/aw-watcher-afk/blob/master/aw_watcher_afk/afk.py#L63

So this could be solved by:

```
git diff                                                                                                                    master 
diff --git a/aw_watcher_mpv_sender/sender.py b/aw_watcher_mpv_sender/sender.py
index dc3bb1f..181aaa6 100644
--- a/aw_watcher_mpv_sender/sender.py
+++ b/aw_watcher_mpv_sender/sender.py
@@ -122,5 +122,6 @@ class Sender:
         """
         key = (bucket_id, event_type)
         if key not in self.buckets_created:
+            time.sleep(1)
             self.client.create_bucket(bucket_id, event_type)
             self.buckets_created.add(key)
```

**Comment by RundownRhino** [comment]
Fixed by #4
